### ISSUE TITLE
0.9.3 contains 'conda' script

### ISSUE THREAD

**Issue by nicoddemus** [conda-forge/conda-devenv-feedstock#3]
All packages for version `0.9.3` contain a `conda` script in the `bin` directory (`Scripts` on Windows):

```
λ ls conda-devenv-0.9.3-py27_0\
bin  info  lib

λ ls conda-devenv-0.9.3-py27_0\bin\
conda  conda-devenv
```

@jakirkham any idea how this might have happened? The recipe didn't really change between `0.9.2` and `0.9.3`.

cc @tadeu @gqmelo

**Comment by nicoddemus** [comment]
Further investigation suggests that this is a bug in `conda-build 2.1.9`.

xref: conda/conda-build#1916

**Comment by jakirkham** [comment]
Sorry. Somehow the issue slipped right through my notifications. 😳

That said, I agree with your conclusion. `conda` does install scripts like this in each environment. It is likely `conda-build` that errored by packaging these instead of disregarding them. Did any other things like this get packaged by accident?

**Comment by nicoddemus** [comment]
> Sorry. Somehow the issue slipped right through my notifications. 

No worries at all!

> Did any other things like this get packaged by accident?

Don't think so, here's the full listing of the files inside one of the linux packages:

```
d:\Downloads\conda-devenv-0.9.3-py27_0\bin
d:\Downloads\conda-devenv-0.9.3-py27_0\info
d:\Downloads\conda-devenv-0.9.3-py27_0\lib
d:\Downloads\conda-devenv-0.9.3-py27_0\bin\conda
d:\Downloads\conda-devenv-0.9.3-py27_0\bin\conda-devenv
d:\Downloads\conda-devenv-0.9.3-py27_0\info\about.json
d:\Downloads\conda-devenv-0.9.3-py27_0\info\files
d:\Downloads\conda-devenv-0.9.3-py27_0\info\git
d:\Downloads\conda-devenv-0.9.3-py27_0\info\has_prefix
d:\Downloads\conda-devenv-0.9.3-py27_0\info\index.json
d:\Downloads\conda-devenv-0.9.3-py27_0\info\LICENSE.txt
d:\Downloads\conda-devenv-0.9.3-py27_0\info\paths.json
d:\Downloads\conda-devenv-0.9.3-py27_0\info\recipe
d:\Downloads\conda-devenv-0.9.3-py27_0\info\recipe\meta.yaml
d:\Downloads\conda-devenv-0.9.3-py27_0\info\recipe\meta.yaml.template
d:\Downloads\conda-devenv-0.9.3-py27_0\lib\python2.7
d:\Downloads\conda-devenv-0.9.3-py27_0\lib\python2.7\site-packages
d:\Downloads\conda-devenv-0.9.3-py27_0\lib\python2.7\site-packages\conda_devenv
d:\Downloads\conda-devenv-0.9.3-py27_0\lib\python2.7\site-packages\conda_devenv-0.9.3-py2.7.egg-info
d:\Downloads\conda-devenv-0.9.3-py27_0\lib\python2.7\site-packages\conda_devenv\.editorconfig
d:\Downloads\conda-devenv-0.9.3-py27_0\lib\python2.7\site-packages\conda_devenv\devenv.py
d:\Downloads\conda-devenv-0.9.3-py27_0\lib\python2.7\site-packages\conda_devenv\devenv.pyc
d:\Downloads\conda-devenv-0.9.3-py27_0\lib\python2.7\site-packages\conda_devenv\_version.py
d:\Downloads\conda-devenv-0.9.3-py27_0\lib\python2.7\site-packages\conda_devenv\_version.pyc
d:\Downloads\conda-devenv-0.9.3-py27_0\lib\python2.7\site-packages\conda_devenv\__init__.py
d:\Downloads\conda-devenv-0.9.3-py27_0\lib\python2.7\site-packages\conda_devenv\__init__.pyc
d:\Downloads\conda-devenv-0.9.3-py27_0\lib\python2.7\site-packages\conda_devenv-0.9.3-py2.7.egg-info\dependency_links.txt
d:\Downloads\conda-devenv-0.9.3-py27_0\lib\python2.7\site-packages\conda_devenv-0.9.3-py2.7.egg-info\entry_points.txt
d:\Downloads\conda-devenv-0.9.3-py27_0\lib\python2.7\site-packages\conda_devenv-0.9.3-py2.7.egg-info\not-zip-safe
d:\Downloads\conda-devenv-0.9.3-py27_0\lib\python2.7\site-packages\conda_devenv-0.9.3-py2.7.egg-info\PKG-INFO
d:\Downloads\conda-devenv-0.9.3-py27_0\lib\python2.7\site-packages\conda_devenv-0.9.3-py2.7.egg-info\requires.txt
d:\Downloads\conda-devenv-0.9.3-py27_0\lib\python2.7\site-packages\conda_devenv-0.9.3-py2.7.egg-info\SOURCES.txt
d:\Downloads\conda-devenv-0.9.3-py27_0\lib\python2.7\site-packages\conda_devenv-0.9.3-py2.7.egg-info\top_level.txt
```

**Comment by jakirkham** [comment]
Yeah AFAICT it looks fine.

This might be a little dicey, but you could try adding a hack to remove `conda` at the end of the build. If it messes up, expect it will be a crash due to trying to remove a file that was already removed. Don't think there are any other unwanted side effects. This should help avoid packaging the script in the interim.

**Comment by nicoddemus** [comment]
> This might be a little dicey, but you could try adding a hack to remove conda at the end of the build. If it messes up, expect it will be a crash due to trying to remove a file that was already removed.

Thanks for the suggestion. I will wait for comments on the issue reported in conda-build first, and depending if a fix will take a while to land I will then push a workaround like the one you suggest. 👍

**Comment by gqmelo** [comment]
@jakirkham, could you remove or mark as broken the buggy packages that were published (all 0.9.3)? They will totally break a miniconda installation as conda-devenv is meant to be installed in the root env.
